### PR TITLE
NRI ERT Modernisation mini-DLC, Part 1

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
This PR is meant to polish and finish whatever I couldn't in the previous one due to over-exhaustion.
Added a nerfed down version of NRI assault pack to Cargo, for whatever LARPing need you'd have.
Made NRI's Szabo-Ivanek service pistol have a 3-burst fire mode, similarly to APS; removed it from Cargo, now being replaced with a Makarov.
Fixed NRI's turrets and viscerators not attacking xenomorphs and the Syndicate.
Also adds a cool """lore""" starting text prompt whenever NRI ERT spawns in.
image
We have also added Teshari-related restrictions to most, if not all of their equipment; shortened ID assignments to use the acronym instead of the full name, and... I think we've added a corpsman reskin for the older armor.
Adds Voentorg goodie pack that's basically a buyable and expensive version of the old NRI gear, as well as edits russian surplus crate to be more uuh... orthodox?
Also, lots of clothing flag-related fixes.

## How This Contributes To The Skyrat Roleplay Experience
Polished updates good(!)
This backpack looks really awesome, I'd assume people'd like to wear them on station.
Szabo-Ivanek was initially meant to have a burst fire mode, but I forgot. :skull:
Turrets and manhacks actually properly working is good.
Lore is also good.

## Changelog

:cl: Stalkeros, Lukaster42 for existing and screaming at me the whole time
add: Adds a "weaker" version of a standard NRI backpack for all your LARPing needs, available in hacked Cargo's Russian crate.
balance: Szabo-Ivanek service pistol now has a 3-burst fire mode. As a result, it's no longer available in Cargo, being replaced with a Makarov.
fix: NRI turrets and viscerators now properly react to threats.
spellcheck: Edits a bunch of miscellaneous weapon descriptions related to NRI.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/novaya_ert/code/ert.dm [hunk 1]
```diff
@@ -21,26 +23,46 @@
...
 	H.grant_language(/datum/language/neorusskya)
+	H.grant_language(/datum/language/schechi)
 
 	return TRUE
 
+/datum/antagonist/ert/nri/greet()
+	if(!ert_team)
+		return
+
+	to_chat(owner, "<span class='warningplain'><B><font size=3 color=yellow>You are the [name].</font></B></span>")
```
Comment (Zonespace27) [inline modular_skyrat/modules/novaya_ert/code/ert.dm]: span defines

File: modular_skyrat/modules/novaya_ert/code/ert.dm [hunk 2]
```diff
@@ -21,26 +23,46 @@
...
+	missiondesc += "<BR><B><font size=3 color=red>You are NOT a deathsquad. You are here to assist the station's Security and Command with alien infestations or enemy activity.</font></B>"
+	missiondesc += "<BR>You are responding to emergency calls from the station for immediate NRI military intervention!\n"
+	missiondesc += "<BR><B>Your Mission</B>:"
+	missiondesc += "<BR> <B>1.</B> Minimise civillian casualties."
+	missiondesc += "<BR> <B>2.</B> Neutralise the threat."
+	missiondesc += "<BR> <B>3.</B> Stabilise the situation."
+	missiondesc += "<BR> <B>4.</B> Contact whoever called you and assist in resolving the matter."
+	missiondesc += "<BR> <B>5.</B> Protect, ensure, and uphold the rights of Novaya Rossiyskaya Imperiya, Kingdom of Agurkrral and Free Teshari League citizens on board [station_name()]. SolFed citizens' wellbeing is up to your discretion."
+
+	missiondesc += "<span class='warningplain'><BR><B>Your Mission</B> : [ert_team.mission.explanation_text]</span>"
```
Comment (Zonespace27) [inline modular_skyrat/modules/novaya_ert/code/ert.dm]: ditto

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/novaya_ert/code/ert.dm (hunk 1 above)
Comment (Stalkeros2) [reply modular_skyrat/modules/novaya_ert/code/ert.dm]: Done.

File: modular_skyrat/modules/novaya_ert/code/ert.dm (hunk 2 above)
Comment (Stalkeros2) [reply modular_skyrat/modules/novaya_ert/code/ert.dm]: Done.

**Comment by Stalkeros2** [comment]
Waiting for actual police sprites to come out. Apparently, the ones I've used were admiral's.

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm [hunk 3]
```diff
@@ -558,6 +558,11 @@ GLOBAL_LIST_INIT(loadout_helmets, generate_loadout_items(/datum/loadout_item/hea
 	item_path = /obj/item/clothing/head/sec/peacekeeper/sol
 	restricted_roles = list(JOB_SECURITY_OFFICER, JOB_SECURITY_MEDIC, JOB_HEAD_OF_SECURITY, JOB_WARDEN)
 
+/datum/loadout_item/head/nriofficercap
+	name = "Security Officer's NRI Cap"
+	item_path = /obj/item/clothing/head/beret/sec/nri
+	restricted_roles = list(JOB_SECURITY_OFFICER, JOB_SECURITY_MEDIC, JOB_HEAD_OF_SECURITY, JOB_WARDEN)
+
```
Comment (ghost) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm]: Security is hired by Lopland and armored by Amradyne, both jobs that operate within SolFed space. Why would they ever think to allow their staff to wear anti-SolFed uniforms, in SolFed space.?

Comment (Stalkeros2) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm]: Oh right I forgot.

**Comment by Stalkeros2** [comment]
Waiting for the actual NRI policemen sprites.

**Comment by Lukaster42** [comment]
No Security stuff for this PR.

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm (hunk 3 above)
Comment (Stalkeros2) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm]: Doesn't exist anymore tbh.

**Comment by Stalkeros2** [comment]
I really don't think that's because of us.

**Pull request merged**